compiler-testing/build.gradle.kts

```diff
@@ -0,0 +1,59 @@
+plugins {
+  id("conventions.library")
+  id("conventions.publish")
+  alias(libs.plugins.kotlin.kapt)
+  alias(libs.plugins.buildconfig)
+}
+
+val VERSION_NAME: String by project
+
+buildConfig {
+  className("BuildProperties")
+  packageName("com.squareup.anvil.compiler.testing")
+  useKotlinOutput { topLevelConstants = true }
+
+  buildConfigField("FULL_TEST_RUN", libs.versions.config.fullTestRun.map { it.toBoolean() })
+  buildConfigField("anvilVersion", VERSION_NAME)
+  buildConfigField("kotlinVersion", libs.versions.kotlin)
+}
+
+conventions {
+  kotlinCompilerArgs.addAll(
+    // The flag is needed because we extend an interface that uses @JvmDefault and the Kotlin
+    // compiler requires this flag when doing so.
+    "-Xjvm-default=all",
+    "-opt-in=org.jetbrains.kotlin.compiler.plugin.ExperimentalCompilerApi",
+  )
+  addTasksToIdeSync("generateBuildConfig")
+}
+
+publish {
+  configurePom(
+    artifactId = "compiler-k2-testing",
```

Comment:
Should the module name + build config package name also reflect this? Current name could cause confusion as being non-K2 or supporting both K1 and K2

Reply:
The artifact id was out of date.  It's updated now to `compiler-testing`.